lib/itamae/resource/remote_directory.rb

```diff
@@ -9,6 +9,8 @@ class RemoteDirectory < Base
       define_attribute :mode, type: String
       define_attribute :owner, type: String
       define_attribute :group, type: String
+      define_attribute :recursive_mode, type: [TrueClass, FalseClass], default: false
+      define_attribute :recursive_owner, type: [TrueClass, FalseClass], default: false
```

Comment:
From the name of `recursive_owner`, users may not expect that group will be also changed recursively.
How about change the name to `recursive_owner_and_group`?